### PR TITLE
Epic 7 - Max Vorobets

### PR DESCRIPTION
**Pull request by vorobmaks**

### CODE REVIEW

**Review by ihoryatsyshyn — APPROVED** [review]
looks good

File: ai_12/maksym_vorobets/Epic7/epic_7_practice_part_maksym_vorobets.cpp [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+    for (int i = 0; i < MATRIX_SIZE; ++i) {
+        for (int j = 0; j < MATRIX_SIZE; ++j) {
+            matrix[i][j] = getRandomNumber(); // 7 в коді використаний двовимірний масив 
+        }
+    }
+}
+
+// Функція для обчислення сум 3-4 випадкових елементів кожного стовпця та створення масиву сум
+int* calculateColumnSums(int matrix[MATRIX_SIZE][MATRIX_SIZE]) {
```
Comment (fm2lpnu) [inline ai_12/maksym_vorobets/Epic7/epic_7_practice_part_maksym_vorobets.cpp:26]: Mistmatch between code and comment

File: ai_12/maksym_vorobets/Epic7/epic_7_practice_part_maksym_vorobets.cpp [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+const int MATRIX_SIZE = 10;
+const char FILE_NAME[] = "output.txt";
+
+// Функція для генерації випадкового числа в межах [0, 100]
+int getRandomNumber() {
+    return rand() % 101;
+}
+
+// Функція для генерації матриці з випадкових чисел
+void generateMatrix(int matrix[MATRIX_SIZE][MATRIX_SIZE]) {
```
Comment (fm2lpnu) [inline ai_12/maksym_vorobets/Epic7/epic_7_practice_part_maksym_vorobets.cpp:15]: Task 1 (a,b) is not implemented according to requirements.
One of two options is implemented

File: ai_12/maksym_vorobets/Epic7/epic_7_practice_part_maksym_vorobets.cpp [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+#include <fstream>
+#include <cstdlib>
+#include <ctime>
+
+const int MATRIX_SIZE = 10;
+const char FILE_NAME[] = "output.txt";
+
+// Функція для генерації випадкового числа в межах [0, 100]
+int getRandomNumber() {
+    return rand() % 101;
```
Comment (fm2lpnu) [inline ai_12/maksym_vorobets/Epic7/epic_7_practice_part_maksym_vorobets.cpp:11]: Not according to requirements.
The math function had to be taken from vns_practice tasks from Epic 7

**Review by fm2lpnu — APPROVED** [review]

**Pull request merged**